tools/gen-device-avr.py

```diff
@@ -183,10 +190,15 @@ def writeGo(outdir, device):
                     out.write(': {description}'.format(**register))
                 out.write('\n')
             for bitfield in register['bitfields']:
-                out.write('\t{name} = 0x{value:x}'.format(**bitfield))
-                if bitfield['description']:
-                    out.write(' // {description}'.format(**bitfield))
-                out.write('\n')
+                name = bitfield['name']
+                value = bitfield['value']
+                bitshift = getShift(value)
+                for c in range(bin(value >> bitshift).count("1")):
+                    v = hex(1 << (bitshift + c))
+                    out.write('\t{0}{1} = {2}'.format(name, c, v))
+                    if bitfield['description']:
+                        out.write(' // {description}'.format(**bitfield))
+                    out.write('\n')
```

Comment:
I think it's best to keep the old behavior when there is only one bit in the bitfield, so that src/runtime/runtime_avr.go doesn't have to be modified. This is more consistent with the AVR header files which don't append the suffix in that case. An easy way to check how many bits there are in a number is using `'{:08b}'.format(value).count('1')`.

Regarding the bit-suffix part: the  for loop can be simplified to:
```python
n = 0
for i in range(8):
    if (value >> i) & 1 == 0: continue
    out.write('\t{}{} = 0x{:x}'.format(name, n, 1 << i))
    if bitfield['description']:
        out.write(' // {description}'.format(**bitfield))
    n += 1
```
This doesn't need the `getShift()` function.

Reply:
Seems reasonable, changes made. 

Note that `UCSR0C_UCSZ00` actually did have 2 values, so that line in `runtime_avr.go` had to be changed to be consistent with the new naming.